corymsmith/react-native-fabric
# [Crashlytics] chore: noticket - merge `master` into `twitterkit-support`

## Summary

The branch that we use for Fabric-driven authorisation for Crowds mobile, `twitterkit-support`, has an incomplete [Crashlytics API (e.g. `crash()`, `recordError()`, `logException()` missing)](https://github.com/conversocial/react-native-fabric#crashlytics-usage).

To get the latest Crashlytics API in the `twitterkit-support`, we merge `master` into `twitterkit-support`.

## How to Test this PR

### QA

In your local repo of `conv_crowds_react`:

1. Ensure `conv_crowds_react` project points to the `chore-noticket-merge-master-into-twitterkit-support` branch, which merged `master` into the `twitterkit-support` branch. In `package.json`:
    ```"react-native-fabric": "git://github.com/conversocial/react-native-fabric.git#chore-noticket-merge-master-into-twitterkit-support"```
2. Install the new dependency: `npm install`
3. Fire up the VM and run `react-native run-android`
4. Sign in with Twitter

**Expected:** App authorisation should succeed and you should see the questions feed.
